toolbox

```diff
@@ -322,7 +322,28 @@ create_enter_command()
 
 
 create_environment_options()
-{
+(
+    columns=""
+    lines=""
+
+    if terminal_size=$(stty size 2>&3); then
```

Comment:
I missed that just `stty(1)` isn't enough when `toolbox(1)` is invoked from inside a container. In that case `stty(1)` will get run against the inner pseudo-terminal pair created by `podman exec --tty` which doesn't have the right sizes. I think we should forward the `COLUMNS` and `LINES` variables from the outer environment created by `toolbox(1)`.